### DISCUSSION TITLE
Compatibility with OrangePi Zero

### DISCUSSION THREAD

**Discussion by eagleDiego** [ketilmo/balena-ads-b#45] [Q&A] [answered]
Hi!

I would like to report that I am successfully running this code on an [OrangePi Zero](http://www.orangepi.org/orangepizero/).
At the moment I only tested the feeding service `adsb-exchange`, but I don't see a reason why the others should not work, since the radio side is working.

I hope this helps to further expand the list of [Supported Devices](https://github.com/ketilmo/balena-ads-b#supported-devices).

Thanks for the great work!

**Comment by ketilmo** [comment, marked as answer]
Hi @eagleDiego,

Thanks for reporting this and for your encouraging feedback. It’s highly appreciated and motivating. It's thanks to the help from people like you we can verify what devices the solution is working on and not.

It’s always good to give credit where it’s due, so feel free to add the device to the readme through a pull request if you like. If not, it’s fine too. Then I will make the addition directly.

Thanks for using balena-ads-b! I wish you a smooth experience onwards! Don't hesitate to reach out again if you have suggestions, comments or questions.

**Reply by ketilmo** [reply]
I have added this as #47

**Reply by eagleDiego** [reply]
Can you please edit the readme from your side? I cannot get to the right icon for OPiZ.

Thank you!

**Reply by ketilmo** [reply]
Hi @eagleDiego,

Sorry for the delayed response. I did try to fetch the correct icon, but apparently, balena has modified their implementation slightly since I did this the last time. I will have a new look at it asap, but things have been hectic here lately.

Did you have the chance to check if all the services run well on the OrangePI?

**Reply by ketilmo** [reply]
Sorry for this taking a while, @eagleDiego! The Orange Pi Zero is finally listed as a supported device on the site. Thanks for verifying!
